storage/range.go

```diff
@@ -1524,7 +1524,7 @@ func (r *Range) loadLastIndex() error {
 // Entries implements the raft.Storage interface.
 // TODO(bdarnell): consider caching for recent entries, if rocksdb's builtin caching
 // is insufficient.
-func (r *Range) Entries(lo, hi uint64) ([]raftpb.Entry, error) {
+func (r *Range) Entries(lo, hi, maxBytes uint64) ([]raftpb.Entry, error) {
```

Comment:
Add a comment about maxBytes being advisory and the function always returning at least one entry.